src/site/content/en/fast/apply-instant-loading-with-prpl/index.md

```diff
@@ -38,26 +38,28 @@ For more information, see [Discover performance opportunities with Lighthouse](/
 
 ## Preload critical resources
 
-Lighthouse shows the following failed audit if a certain resource is parsed and
+Lighthouse shows the following failed audit if a late-discovered resource is parsed and
```

Comment:
Don't think this change is needed as covered by the "and fetched late" part later.